### PR TITLE
Show collapsible analyses results

### PR DESCRIPTION
**Pull request by charisk**
Created a re-usable CollapsibleItem component and used that to show the results of analyses.

## Checklist
N/A
- [ ] [CHANGELOG.md](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/CHANGELOG.md) has been updated to incorporate all user visible changes made by this pull request.
- [ ] Issues have been created for any UI or other user-facing changes made by this pull request.
- [ ] _[Maintainers only]_ If this pull request makes user-facing changes that require documentation changes, open a corresponding docs pull request in the [github/codeql](https://github.com/github/codeql/tree/main/docs/codeql/codeql-for-visual-studio-code) repo and add the `ready-for-doc-review` label there.

### CODE REVIEW

File: extensions/ql-vscode/src/remote-queries/view/CollapsibleItem.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  children: React.ReactNode
+}) => {
+  const [isExpanded, setExpanded] = useState(false);
+  return (
+    <>
+      <Container>
+        <Button onClick={() => setExpanded(!isExpanded)}>
+          {isExpanded
+            ? <ChevronUpIcon size={16} />
+            : <ChevronDownIcon size={16} />
```
Comment (shati-patel) [inline extensions/ql-vscode/src/remote-queries/view/CollapsibleItem.tsx:37]: Looking at the design, I think we want "ChevronDown" for the expanded list 🔽 
I can't see a specific design for the collapsed list, but probably "ChevronRight" ▶️ That seems fairly standard (e.g. in the Files Changed tab here on GH 😄)

**Review by shati-patel — COMMENTED** [review]
Looks good ✨ Small comment, from checking this out locally! There's also a code scanning alert inline 🔍

File: extensions/ql-vscode/src/remote-queries/view/CollapsibleItem.tsx (hunk 1 above)
Comment (charisk) [reply extensions/ql-vscode/src/remote-queries/view/CollapsibleItem.tsx]: Thanks, I thought it looked a odd! 😅

**Review by shati-patel — APPROVED** [review]
LGTM ✨

**Pull request merged**